rename.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+charm=$( awk '/^name:/{ print $2 }' < metadata.yaml )
```

Comment:
I understand that we've been using similar logic to devise the charm name in other Ceph charms as well but since this always comes to be `microceph` can we not just name it  `microceph.charm` directly ? I am obviously assuming the name `microceph` will not change in future.

Reply:
This is just for discussion BTW, I am approving this change.